sootup.core/src/main/java/sootup/core/signatures/MethodSubSignature.java

```diff
@@ -134,4 +134,22 @@ public void toString(StmtPrinter printer) {
 
     printer.literal(")");
   }
+
+  boolean isMainSignature() {
```

Comment:
1. Do you really want to have Java-specific methods in the core?
2. Should these be public?

Reply:
there was something.. other ~~unsupported~~ possible languages.. we can have another structure like JavaLanguage where  we can store this, (java.base/)`java.lang.throwable`, `java.lang.Object` etc.?

Reply:
it affects Identifiers moved it to the JavaIdentifierFactory